dvc/command/checkout.py

```diff
@@ -34,6 +35,12 @@ def add_parser(subparsers, parent_parser):
         default=False,
         help="Do not prompt when removing working directory files.",
     )
+    checkout_parser.add_argument(
+        "--relink",
+        action="store_true",
+        default=False,
+        help="Relink all files to cache, including unchanged.",
```

Comment:
cc @jorgeorpinel

Reply:
```suggestion
        help="Link/copy (overwrite) all files from cache to workspace again.",
```
Not sure about "copy" part, depends what this actually does.